kingpin/actors/rightscale/server_array.py

```diff
@@ -1326,6 +1326,9 @@ def _execute_array_with_concurrency(self, arrays, inputs):
             inputs: A string of inputs generated by
```

Comment:
I'd just update the docstring as well, since it will also accept a non-list type with this change.